recipes/ctags/bld.bat

```diff
@@ -0,0 +1,4 @@
+nmake -f mk_mvc.mak
```

Comment:
Those instructions are pretty Python-oriented and this recipe has nothing to do with Python, so I took a guess and added both the features and build blocks but no python dependency (https://github.com/conda-forge/staged-recipes/pull/3207/commits/2fa59b80bc35c2b4d21e7e99d60ab6234641c4eb). Was that right?